disable_odoo_online/data/ir_cron.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="mail.ir_cron_module_update_notification" model="ir.cron">
+        <field name="active" eval="False" />
```

Comment:
What this does?

Reply:
This used to deactivate function `update_notification` in module mail. Function `update_notification` used to "Send a message to Odoo's publisher warranty server to check the validity of the contracts, get notifications, etc...".

Reply:
Hmmmm I thought we had to leave this active, and instead disabled it in `PublisherWarrantyContract.update_notification`. Am I remembering incorrectly @legalsylvain?